src/site/content/en/blog/gedi/index.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title: |
+  GEDI Digital reduced mobile bounce rate by 8% with Core Web Vitals
+subhead: |
+  By focusing development effort on performance improvement and applying the
+  latest web technologies and techniques, GEDI Digital has created one of the
+  fastest newspaper sites.
+authors:
+  - maurizioparadisi
+date: 2021-12-10
+hero: image/VbsHyyQopiec0718rMq2kTE1hke2/qg8rHQ5qlKK9t4Z6qTH2.jpeg
+description: |
+  By focusing development effort on performance improvement and applying the
+  latest web technologies and techniques, GEDI Digital has created one of the
+  fastest newspaper sites.
+tags:
+  - blog
+  - case-study
+  - web-vitals
+---
+
+## Introduction
+
+Based in Rome, Italy, and formed in 1976, [GEDI](http://www.gedispa.it/) is a
+media company that publishes 12 newspapers, several magazines, radio stations,
+and 15 news sites. Their stable of publications includes
+[La Stampa](https://www.lastampa.it/) and
+[La Repubblica](https://www.repubblica.it/).
+
+While GEDI is present in traditional publishing and broadcasting, its content
+is increasingly being consumed on the web, and web is now contributing
+significantly to their advertising revenue. Therefore, user retention and user
+experience on the web are vital to the company’s overall prosperity.
+
+## Challenge
+
+According to Giuseppe Covato, Head of Innovation and New Technologies at GEDI
+Digital, before 2017, they didn't have a strategic path for web performance.
+Knowledge sharing between teams was minimal and inefficient. In particular, it
+was hard to get marketing on board on technical topics such as Core Web Vitals.
+
+With the introduction of Rep, a new premium content site, the Digital
+team took the opportunity to experiment with emerging technologies such as
+Polymer, AMP, and [progressive web apps](/progressive-web-apps/) (PWA). The
+project also served as a playground to upscale the team around
+performance-related objectives.
+
+## Optimizations
+
+To address performance, the team first applied critical rendering path
+optimization, making sure there was no CSS or JavaScript that could slow down
```

Comment:
```suggestion
To address performance, the team first applied [critical rendering path
optimization](https://developers.google.com/web/fundamentals/performance/critical-rendering-path/optimizing-critical-rendering-path), making sure there was no CSS or JavaScript that could slow down
```

Reply:
I know that this is from WebFundamentals, but this is a pretty good piece to link to. I'm sure it will make it in the migration to web.dev at some point (possibly when I create the performance course next year), so it will eventually get redirected as well.